### ISSUE TITLE
cabal install --ghcjs pointed fails with Data.Functor.Bind isn't safe

### ISSUE THREAD

**Issue by btrepp** [ghcjs/ghcjs#621]
```
cabal install --ghcjs pointed                                                                        
Resolving dependencies...                                                                              
Configuring pointed-5...                                                                               
Building pointed-5...                                                                                  
Failed to install pointed-5                                                                            
Build log ( C:\Users\Beau\AppData\Roaming\cabal\logs\ghcjs-0.2.0\pointed-5-LSb06HrTiFrA86OcrYuFq5.log )
:                                                                                                      
Preprocessing library for pointed-5..                                                                  
Building library for pointed-5..                                                                       
[1 of 2] Compiling Data.Copointed   ( src\Data\Copointed.hs, dist\build\Data\Copointed.js_o )          
                                                                                                       
src\Data\Copointed.hs:31:1:                                                                            
    Data.Functor.Bind: Can't be safely imported!                                                       
    The module itself isn't safe.                                                                      
cabal: Leaving directory 'C:\Users\Beau\AppData\Local\Temp\cabal-tmp-2336\pointed-5'                   
cabal: Error: some packages failed to install:                                                         
pointed-5-LSb06HrTiFrA86OcrYuFq5 failed during the building phase. The                                 
exception was:                                                                                         
ExitFailure 1                   
```                          
```
 ghcjs --version
The Glorious Glasgow Haskell Compilation System for JavaScript, version 0.2.0 (GHC 7.10.2)
```

OS is windows 10.

**Comment by luite** [comment]
Hmm, the safety of some modules differs from GHC, probably because some patches for the boot libraries. One difference somewhere high up can affect inferred safety of many things. We should probably verify the safety status of the modules in packages bundled with GHCJS (built by `ghcjs-boot`) and keep it in sync with upstream as much as possible.

I'm not sure if there's a workaround here.

**Comment by luite** [comment]
this should be fixed in the `ghc-8.2` branch
